### DISCUSSION TITLE
Add OpenRCT2 Intro

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17732] [Feature requests]
I think it would be nice if the community or the devs put together an intro to replace the stock RCT2 Vanilla intro that highlights some great features or parks from openrct2

**Comment by karst** [comment]
This is already done, we have done this a few times actually. Most recent community made intro is 0.4.0's.

**Comment by 733737** [comment]
I mean the "Infogrames" "Chris Sawyer" "RollerCoaster Tycoon 2" Intro, not park sequences

**Reply by karst** [reply]
You can enable this in config.ini in your documents folder

**Reply by 733737** [reply]
yes I know that, I was thinking it could say OpenRCT2 and feature some parks instead of being RCT2 on a white background

**Reply by Lastorder-DC** [reply]
That is legacy behavior and might be removed in future release.
There are no publisher/dev company logo to show in OpenRCT2, so logo screen is also not needed.

No need to modify elements are not required.
